backend/endpoints/user_request.py

```diff
@@ -0,0 +1,42 @@
+import logging
+from flask import jsonify
+
+from models import Request
+
+log = logging.getLogger("user_request")
+
+
+def create_request(request_json):
+    """Creates and persists a new user request into database.
+
+    Parameters
+    ----------
+    request_json : dict
+        A dictionary representing the user request details.
+        example {
+                  "beneficiary": 123456,
+                  "user": 123456,
+                  "status": "new",
+                  "secret": "345%443$$",
+                  "urgent": false,
+                  "comments": "string",
+                  "has_symptoms": false,
+                  "cluster": 34,
+                  "created_at": "2020-10-04T21:31:16.741Z"
+                }
+    Returns
+    -------
+    200:
+        If the request was successfully created and saved.
+    400:
+        If the request wasn't created or saved, and there was raised some exception.
+    """
+    try:
+        user_request_data = request_json
+        log.debug(request_json)
+        user_request = Request(**user_request_data)
```

Comment:
```
                  "user": 123456,
                  "created_at": "2020-10-04T21:31:16.741Z"
```
@alexciornii @iamandrewluca is it better to add these fields in the backend? not to be send from frontend?

Reply:
it's not necessary, because this fields are filling automatic

Reply:
@sroy8091  use the` auth.username()` as [here](https://github.com/code4moldova/voluntar-backend/blob/development/backend/app.py#L100)

Reply:
@johnsmithm so basically instead of taking user from body we will populate according to user requesting?

Reply:
@johnsmithm can you also confirm this?

Reply:
yes